edenia/antelope-tools
# Feat/UI improvements 1105

### Apply UI Improvements 

### What does this PR do?

- Resolve #1105
- Fix tooltip in "Rewards distribution" page
- Fix the breakpoints of the homepage
- Refactor code for more info tooltip

### Steps to test

1. Run the project locally
2. Go to "Nodes" page and check the spacing
3. Go to "Accounts and Contracts" page  and check the spacing
4. Go to "Rewards distribution" and check the tooltips

### Screenshots

Align items and use a tooltip to show the producer key
![imagen](https://user-images.githubusercontent.com/66583677/210595645-7e4ed5a9-b318-44b4-b4b7-88396a833b02.png)
![imagen](https://user-images.githubusercontent.com/66583677/210595893-154b80ac-485a-4a95-ac53-43ea7ea8dd16.png)

Fix margins in the "Accounts and contracts" page
![imagen](https://user-images.githubusercontent.com/66583677/210596033-75b007d0-e4b9-44a9-aa96-592b2abbc460.png)
![imagen](https://user-images.githubusercontent.com/66583677/210596249-f9e9a8b9-da3c-470c-a604-23f34de8b05a.png)

Content is centered and measurements are the same as those of the design 
![imagen](https://user-images.githubusercontent.com/66583677/210597490-bb245d3f-a6f1-41e3-ac8a-ce37a5feb919.png)
